dboperations/dboperations_test.go

```diff
@@ -166,6 +166,55 @@ func TestGetVersionAllSuccess(t *testing.T) {
 	}
 }
 
+func TestGetVersionAllSuccessForPackageDetailsModel(t *testing.T) {
```

Comment:
update TestGetVersionAllSuccess with chef-ice scenario instead on creating this testcase

Reply:
Done